Snackr/Views/Shared/_Layout.cshtml

```diff
@@ -24,13 +25,24 @@
               <span class="icon-bar"></span>
               <span class="icon-bar"></span>
           </button>
-          <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">Snackr</a>
+          <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-left"><img class="img1" src="~/images/snackr_logo.png"></a>
       </div>
     <ul class="nav navbar-nav">
         <li><a asp-area="" asp-controller="Snackr" asp-action="Index">Home</a></li>
         <li><a asp-area="" asp-controller="Inventory" asp-action="Index">Inventory</a></li>
-        <li><a asp-area="" asp-controller="Request" asp-action="Index">Request</a></li>
+        @* <li><a asp-area="" asp-controller="Request" asp-action="Index">Request</a></li> *@
```

Comment:
Remove commented code if you aren't going to use it

Reply:
removed.